pyathena/connection.py

```diff
@@ -139,7 +140,6 @@ def __init__(
                 **self._session_kwargs,
             )
 
-        self.config: Optional[Config] = config if config else Config()
         self.config.user_agent_extra = (
```

Comment:
I don't think these `config.user_agent_extra` lines are relevant to create the `sts` client though.
If they are relevant, I can move them also to the lines above.